src/copaw/app/channels/feishu/channel.py

```
@@ -617,6 +619,61 @@ async def _on_message(self, data: "P2ImMessageReceiveV1") -> None:
+                        line_str += item.get("text", "")
+                    elif tag == "a":
+                        text_val = item.get("text", "")
+                        href_val = item.get("href", "")
+                        line_str += f"[{text_val}]({href_val})"
+                    elif tag == "at":
+                        line_str += f"@{item.get('user_id', '')}"
+                    elif tag == "img":
+                        line_str += "[image]"
+                if line_str:
+                    text_parts.append(line_str)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)